pages/integrations/output_framework.rst

```diff
@@ -57,29 +57,36 @@ a processing pipeline which will be executed on each message coming from the sou
 not wish to forward.  It can also be used to add data to modify the contents of the outgoing
```

Comment:
```suggestion
Not wish to forward. It can also be used to add data to modify the contents of the outgoing. 	
```

Reply:
The beginning of this line is not the beginning of a sentence.  Nor is the end of this line the end of a sentence.